src/popup/router/index.ts

```diff
@@ -68,6 +70,17 @@ router.beforeEach(async (to, from, next) => {
     return;
   }
 
+  // Bitcoin accounts can only visit default (transaction-list) view in account-details page
+  if (
+    activeAccount.value.protocol === PROTOCOL_BITCOIN
+    && to.name !== ROUTE_ACCOUNT_DETAILS
+  ) {
+    if (to.matched.some((record) => record.path === `/${ROUTE_ACCOUNT_DETAILS}/`)) {
+      next({ name: ROUTE_ACCOUNT_DETAILS });
+      return;
+    }
+  }
```

Comment:
I think it's bad to have the protocol related logic in a global file. I can imagine that more protocols we have more bloated this file will get.